.github/workflows/docker-openmetadata-server.yml

```diff
@@ -9,48 +9,146 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-name: docker-openmetadata-server docker
+name: docker-openmetadata-server release app
 on:
   workflow_dispatch:
     inputs:
-      tag:
-        description: "Input tag"
+      GITHUB_RELEASE_TAG:
+        description: "Github Release Tag"
+        required: true
+      DOCKER_RELEASE_TAG:
+        description: "Docker Release Tag"
         required: true
   release:
     types: [published]
-
 jobs:
+  maven-build:
+    runs-on: ubuntu-latest
+    steps:
+
+      # - name: Check trigger type
+      #   if: ${{ env.input == '' }}
+      #   run: echo "input=0.13.0" >> $GITHUB_ENV
+      
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      
+      - name: Setup Test Containers Properties
+        run: |
+          sudo make install_antlr_cli
+          echo 'testcontainers.reuse.enable=true' >> $HOME/.testcontainers.properties
+
+      - name: Build PR with Maven
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git fetch --no-tags https://github.com/open-metadata/OpenMetadata.git +refs/heads/main:refs/remotes/origin/main
+          mvn versions:set -DnewVersion=0.13.0
```

Comment:
We shouldn't be doing this here. Let's remove. The assumption here will be the repo source will already have the version number updated!